lib/Minion/Backend/Pg.pm

```diff
@@ -223,18 +223,19 @@ sub retry_job {
 sub stats {
   my $self = shift;
 
-  my $stats = $self->pg->db->query(
-    "SELECT COUNT(*) FILTER (WHERE state = 'inactive' AND (expires IS NULL OR expires > NOW())) AS inactive_jobs,
-       COUNT(*) FILTER (WHERE state = 'active') AS active_jobs, COUNT(*) FILTER (WHERE state = 'failed') AS failed_jobs,
-       COUNT(*) FILTER (WHERE state = 'finished') AS finished_jobs,
-       COUNT(*) FILTER (WHERE state = 'inactive' AND delayed > NOW()) AS delayed_jobs,
-       (SELECT COUNT(*) FROM minion_locks WHERE expires > NOW()) AS active_locks,
-       COUNT(DISTINCT worker) FILTER (WHERE state = 'active') AS active_workers,
-       (SELECT CASE WHEN is_called THEN last_value ELSE 0 END FROM minion_jobs_id_seq) AS enqueued_jobs,
-       (SELECT COUNT(*) FROM minion_workers) AS workers,
-       EXTRACT(EPOCH FROM NOW() - PG_POSTMASTER_START_TIME()) AS uptime
-     FROM minion_jobs"
-  )->hash;
+  my $stats = $self->pg->db->query(qq{
+  SELECT
```

Comment:
Indentation and quoting is still inconsistent, otherwise it looks ok.

Reply:
I see three different quoting formats in the existing module:

- '
- "
- q{}

I can modify to whichever one is preferable. Below is modified to use q{} instead of the current qq{} and I pushed the first line indentation in to match some of the other queries at 4 spaces.

```
sub stats {
  my $self = shift;

  my $stats = $self->pg->db->query(q{
    SELECT
    (SELECT COUNT(*) FROM minion_jobs WHERE state = 'inactive' AND (expires IS NULL OR expires > NOW())) AS inactive_jobs,
    (SELECT COUNT(*) FROM minion_jobs WHERE state = 'active') AS active_jobs,
    (SELECT COUNT(*) FROM minion_jobs WHERE state = 'failed') AS failed_jobs,
    (SELECT COUNT(*) FROM minion_jobs WHERE state = 'finished') AS finished_jobs,
    (SELECT COUNT(*) FROM minion_jobs WHERE state = 'inactive' AND delayed > NOW()) AS delayed_jobs,
    (SELECT COUNT(*) FROM minion_locks WHERE expires > NOW()) AS active_locks,
    (SELECT COUNT(DISTINCT worker) FROM minion_jobs mj WHERE state = 'active') AS active_workers,
    (SELECT CASE WHEN is_called THEN last_value ELSE 0 END FROM minion_jobs_id_seq) AS enqueued_jobs,
    (SELECT COUNT(*) FROM minion_workers) AS workers,
    EXTRACT(EPOCH FROM NOW() - PG_POSTMASTER_START_TIME()) AS uptime
  })->hash;
  $stats->{inactive_workers} = $stats->{workers} - $stats->{active_workers};

  return $stats;
}
```

Reply:
Just make it look exactly like one of the existing cases, including quotation token in front of and after the SQL string.

Reply:
We indent with 2 spaces everywhere.

Reply:
You'll need to let me know if the indentation is how you want it. Since this is not like any other queries in the module, I indented 2 spaces after the initial SELECT statement for each sub-select and the extract, there are no WHERE or FROM statements.

I modified quoting to just use double quotes and wrapped it similarly to the other queries in the module.